spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -676,8 +676,10 @@ def add_electrical_series(
         use_times: bool (optional, defaults to False)
             If True, the times are saved to the nwb file using recording.frame_to_time(). If False (defualut),
             the sampling rate is used.
-        write_as_lfp: bool (optional, defaults to False)
-            If True, writes the traces under a processing LFP module in the NWBFile instead of acquisition.
+        write_as: str (optional, defaults to 'raw')
+            Options: 'raw', 'lfp' or 'processed'
```

Comment:
Can you expand on this, letting the user know what precisely it accomplishes?

Reply:
sure!

Reply:
done